### PR TITLE
Create View

### PR DESCRIPTION
**Pull request by Epaltechs**
### At this juncture in the project, I:
- Implement the design from the sneak peek wireframes. (https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png?raw=true)
- Use methods that were created in the processing data in models.
- Add some data to the database.

### CODE REVIEW

File: db/migrate/20220831160429_create_articles.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateArticles < ActiveRecord::Migration[7.0]
+  def change
+    create_table :articles do |t|
+      t.string :title
+      t.text :body
+
+      t.timestamps
```
Comment (KDlamini) [inline db/migrate/20220831160429_create_articles.rb:7]: When I try to populate the database with data to test your views I get an error, specifically with creating posts.

Screenshot from 2022-08-31 23-22-30

This happens when I try to creating a post by running:
`first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')`

after creating users with;
```
   first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
   second_user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
```

and comments with:
`Comment.create(post: first_post, author: second_user, text: 'Hi Tom!' )`

If you are using a different format to insert data into your database, kindly create sample data in your `db/seed.rb` file which can be run to populate your database.

Please make sure that:

- There's at least one user
- At least 4 posts written by one of the users you created
- At least 6 posts comments for one of the posts you created

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Epaltechs 🙋‍♀️,  

Great work so far! 💯 
You still need to work on some issues to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Good PR tdescription
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Epaltechs** [comment]
#### At this point, I create sample data in db/seeds.rb file that can be run to populate the database, as requested by the code reviewer. Also, I have added a direction in the README file on how to run the code.

**Review by DeliceLydia — APPROVED** [review]
## Hi @Epaltechs 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**